lua/lazy_snapshot.lua

```diff
@@ -41,7 +41,7 @@ return {
   { "rcarriga/nvim-dap-ui", version = "^3", optional = true },
   { "rcarriga/nvim-notify", version = "^3", optional = true },
   { "rebelot/heirline.nvim", commit = "2a151df2dc870e79b138a59ebaaaddf3d1b0d703", optional = true },
-  { "s1n7ax/nvim-window-picker", version = "^1", optional = true },
+  { "s1n7ax/nvim-window-picker", version = "^2", optional = true },
   { "saadparwaiz1/cmp_luasnip", commit = "18095520391186d634a0045dacaa346291096566", optional = true },
   { "stevearc/aerial.nvim", commit = "aff1bb8fecff83d3e3a2d544c4d4e6d65718bd19", optional = true },
   { "stevearc/dressing.nvim", commit = "f19cbd56f7f8cad212c58a7285d09c5d9c273896", optional = true },
```

Comment:
line 48, just curious on why pinned at `1.1`?

Reply:
It is pinned to at least v1.1.  so `[1.1,2)` in math notation